core/frontend/src/components/app/SettingsMenu.vue

```diff
@@ -92,6 +104,20 @@ export default Vue.extend({
           notifier.pushBackError('RESET_SETTINGS_FAIL', error)
         })
     },
+    async remove_service_log_files(): Promise<void> {
+      await back_axios({
+        url: `${API_URL}/services/remove_log`,
```

Comment:
```suggestion
        url: `${API_URL}/settings/remove_log`,
```
![image](https://user-images.githubusercontent.com/4013804/216436210-c90cbbb6-3b4e-4979-9d23-1085f2a63be8.png)

Reply:
ctrl-c issue